### PR TITLE
Install hpcbind script: issue #3304

### PR DESCRIPTION
**Pull request by jjwilke**

### CODE REVIEW

**Review by masterleinad — APPROVED** [review]
Looks good to me.

File: CMakeLists.txt [hunk 1]
```diff
@@ -272,7 +272,13 @@ INCLUDE(${KOKKOS_SRC_PATH}/cmake/kokkos_install.cmake)
...
 # If the argument of DESTINATION is a relative path, CMake computes it
 # as relative to ${CMAKE_INSTALL_PATH}.
-INSTALL(PROGRAMS ${CMAKE_CURRENT_SOURCE_DIR}/bin/nvcc_wrapper DESTINATION ${CMAKE_INSTALL_BINDIR})
+# Also install other utility scripts
+SET(BIN_SCRIPTS
+  nvcc_wrapper
+  hpcbind)
+FOREACH(SCRIPT ${BIN_SCRIPTS})
+  INSTALL(PROGRAMS ${CMAKE_CURRENT_SOURCE_DIR}/bin/${SCRIPT} DESTINATION ${CMAKE_INSTALL_BINDIR})
+ENDFOREACH()
```
Comment (dalg24) [inline CMakeLists.txt:281]: Any reason why you preferred a loop even though the `install` command supports 
```CMake
INSTALL(
  PROGRAMS
    bin/nvcc_wrapper
    bin/hpcbind
  DESTINATION ${CMAKE_INSTALL_BINDIR})
```

**Comment by DavidPoliakoff** [comment]
@crtrott , do we want to maintain these scripts, and promise their effectiveness? If yes, I think this is good, but that's still the question we haven't answered

**Comment by crtrott** [comment]
who is using hpcbind and who is volunteering to maintain it? This was Dan's baby. @nmhamster Si do you think this is a good idea or bad idea (hpcbind)?

**Review by crtrott — CHANGES_REQUESTED** [review]
Just putting a block on this, I want to hear from @nmhamster how he feels about the script.

**Comment by DavidPoliakoff** [comment]
We had a user in Slack request this a month ago or so, I filed the tracking issue and helpfully didn't name them (oops). I'll see if I can find them in our history

File: CMakeLists.txt (hunk 1 above)
Comment (jjwilke) [reply CMakeLists.txt:281]: Because I was trying to fit in work between my child's naps. Let us pretend this didn't happen and never speak of this again.

**Comment by crtrott** [comment]
Talked with nmhamster and he agrees its good to install this. Just need to resolve the merge conflict.

File: cmake/kokkos_tribits.cmake [hunk 2]
```diff
@@ -287,17 +287,22 @@ MACRO(KOKKOS_CONFIGURE_CORE)
 ENDMACRO()
 
 ## KOKKOS_INSTALL_ADDITIONAL_FILES - instruct cmake to install files in target destination.
-##                        Includes generated header files, nvcc_wrapper and
-##                        other files provided through plugins
+##                        Includes generated header files, scripts such as nvcc_wrappera and hpcbind,
```
Comment (jeffmiles63) [inline cmake/kokkos_tribits.cmake:290]:
```suggestion
##                        Includes generated header files, scripts such as nvcc_wrapper and hpcbind,
```

Comment (dalg24) [reply cmake/kokkos_tribits.cmake]: Fixed thank you

**Comment by dalg24** [comment]
Failure is unrelated

**Pull request merged**